src/style.css

```diff
@@ -1,3 +1,73 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
The universal selector '*' is used here to set box-sizing for all elements. This can impact performance, especially on large pages with many elements. Consider setting 'box-sizing: border-box;' only to elements that require it, or use a class to apply it selectively.

Reply:
➕➖